README.md

```diff
@@ -79,10 +79,14 @@ Notably, this deployment will:
   linode-block-storage (default)   linodebs.csi.linode.com   2d
   ```
   
-* use a `reclaimPolocy` of `Released` [Learn More](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/)
+* use a `reclaimPolicy` of `Released` [Learn More](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/)
   
   Volumes created by this CSI driver will automatically be deleted when they are no longer needed.
 
+* assume that the [Linode CCM](https://github.com/linode/linode-cloud-controller-manager) is initialized and running [Learn More](https://kubernetes.io/docs/reference/command-line-tools-reference/cloud-controller-manager/)
+
+  If this is not the case, you must modify the init container script located in the `08-cm-get-linode-id.yaml` ConfigMap and delete [the line](https://github.com/linode/linode-blockstorage-csi-driver/blob/master/pkg/linode-bs/deploy/kubernetes/08-cm-get-linode-id.yaml#L18) that contains the `exit 1`.
```

Comment:
"If this is not the case" -> "If you absolutely intend to run this on a cluster which will not run the Linode CCM"